pcsx2/ps2/Iop/IopHwWrite.cpp

```diff
@@ -170,6 +173,11 @@ static __fi void _HwWrite_16or32_Page1( u32 addr, T val )
 
 	u32 masked_addr = addr & 0x0fff;
 
+	// todo: psx mode: this is new
```

Comment:
same comment as previously use masked_addr and add nice define (or constant) name